zio-interop-cats-tests/shared/src/test/scala/zio/interop/CatsMtlSpec.scala

```diff
@@ -21,6 +21,20 @@ class CatsMtlSpec extends ZioSpecBase {
     implicit tc => LocalTests[ZIO[Ctx, Error, _], ZEnvironment[Ctx]].local[ZEnvironment[Ctx], Int]
   )
 
+  Unsafe.unsafe { implicit unsafe =>
+    implicit val f: FiberRef[Ctx] = FiberRef.unsafe.make("")
```

Comment:
i wasn't sure how best to make a fiber ref in this test suite, any guidance would be appreciated